setup/terminal.md

```diff
@@ -0,0 +1,33 @@
+# Why terminal 
+
+Majority of people when they first start to use the computers they do it through the GUI (Graphical user interface). They do it by clicking or touching things on the screen. 
+This is the best way of interacting with computers today, maybe in near future we will use voice more and more, however this is not always the fastest way of doing things on the computer.
```

Comment:
This is the best way of interacting with computers today, maybe in the near future we will use our voices more and more, however this isn't always the fastest way of doing things on your computer.